tests/common/snappi_tests/common_helpers.py

```diff
@@ -902,7 +902,49 @@ class packet_capture(Enum):
     IP_CAPTURE = "IP_Capture"
 
 
-def config_capture_pkt(testbed_config, port_names, capture_type, capture_name=None):
+class pfc_traffic_flow(Enum):
+    """
+    ENUM of pfc traffic settings
+    CONTINUOUS - -100
+    BURST - -99
+    FIXED_PACKETS - -98
+    FIXED_DURATION - -97
+    """
+    CONTINUOUS = -100
+    BURST = -99
+    FIXED_PACKETS = -98
+    FIXED_DURATION = -97
+
+
+class data_traffic_flow(Enum):
+    """
+    ENUM of data/test traffic settings
+    CONTINUOUS - -100
+    BURST - -99
+    FIXED_PACKETS - -98
+    FIXED_DURATION - -97
+    """
+    CONTINUOUS = -100
+    BURST = -99
+    FIXED_PACKETS = -98
+    FIXED_DURATION = -97
+
+
+class background_traffic_flow(Enum):
```

Comment:
We cannot create a unified class since each traffic type may have different configs ex. pfc pause flow could be configured to continuous mode but data traffic may be configured to burst or fixed packets mode.